src/Parallel/Algorithm.hpp

```diff
@@ -428,8 +432,15 @@ class AlgorithmImpl<ParallelComponent, tmpl::list<PhaseDepActionListsPack...>>
   // handling the cases where the `apply` method returns a tuple of one, two,
   // or three elements, in order:
   // 1. A DataBox
-  // 2. A bool determining whether or not to terminate (and potentially move
-  //   to the next phase)
+  // 2. Either:
+  //    2a. A bool determining whether or not to terminate (and potentially move
+  //        to the next phase), or
+  //    2b. An `AlgorithmExecution` object describing whether to continue,
+  //        pause, or halt. `AlgorithmExecution::Continue` is equivalent to
+  //        passing `false` for termination, `AlgorithmExecution::Pause` is
```

Comment:
In the implementation, returning false sets `terminate_` to false, but `Continue` doesn't do anything, which is different if `terminate_` was not already false.  I don't know which is intended, but I personally would find the "do nothing" behavior useful.

Reply:
That's a good point. For now, I think I'll keep the functionality of leaving the termination flag unchanged for `Continue`, and modify the comments to reflect that behavior.